packages/blocks/src/_common/print-manager/print-manager.ts

```diff
@@ -0,0 +1,124 @@
+export class PrintManager {
```

Comment:
I think we should put this in `@blocksuite/affine-shared/utils`.

Reply:
done